test/node/request-api.js

```diff
@@ -25,34 +24,7 @@ describe('\'deal with HTTP weirdness\' tests', () => {
   })
 })
 
-describe('trailer headers', () => {
```

Comment:
I know these do not work in browsers (can't read value via JS: https://github.com/mdn/browser-compat-data/issues/14703) but are they also broken in Node?

Reply:
Will look into this now

Reply:
yes, broken in node. 

1. `.add(stuff, callback)` is invalid. callback doesn't seem to be supported.
2. switching to await and getting response provides empty response.
3. request is never actually ended, hence the timeout

looking into this a bit before switching back to skipped